when-parallel-dependent.yaml

```diff
@@ -2,18 +2,19 @@ version: 1.0-preview-1
 steps:
   # build website and func-test images, concurrently
   - id: build-hello-world
-    build: -t {{.Run.Registry}}/hello-world:{{.Run.ID}} .
+    build: -t {{.Run.Registry}}/hello-world:{{.Run.ID}} -f hello-world.dockerfile .
     when: ["-"]
   - id: build-hello-world-test
-    build: -t hello-world-test .
+    build: -t hello-world-test -f hello-world.dockerfile .
```

Comment:
Same mismatch - building hello-world-test here, but calling hello-world-func-test in the `cmd` below.